Convert-ProwlarrSupportedIndexersToMarkdownTable.ps1

```diff
@@ -1,108 +1,168 @@
-#Requires -Module FormatMarkdownTable
+﻿#Requires -Module FormatMarkdownTable
 
 <#
     .SYNOPSIS
         Name: Convert-ProwlarrSupportedIndexersToMarkdownTable.ps1
         The purpose of this script is to export a markdown table for the wiki of the available indexers
-
     .DESCRIPTION
-        Grabs available indexers from a locally installed Prowlarr instance
-
+        Grabs build number and available indexers from a local or remotely installed Prowlarr instance
     .NOTES
-        This script has been tested on Windows PowerShell 5.1 and PowerShell 7.1.3
-
+        This script has been tested on Windows PowerShell 7.1.3
     .EXAMPLE
-    PS> .\Convert-ProwlarrSupportedIndexersToMarkdownTable.ps1 -Commit 1.1.1.1 -Build "test" -APIKey "asjdhfjashdf89787asdfsad87676" -BaseURL http://prowlarr:9696 -OutputFile "supported-indexers.md"
-    
+    PS> .\Convert-ProwlarrSupportedIndexersToMarkdownTable.ps1 -Commit 1.1.1.1 -Build "test" -AppAPIKey "asjdhfjashdf89787asdfsad87676" -AppBaseURL http://prowlarr:9696 -OutputFile "supported-indexers.md"
     .EXAMPLE
-    PS> .\Convert-ProwlarrSupportedIndexersToMarkdownTable.ps1 -Commit 1.1.1.1 -Build "test" -APIKey "asjdhfjashdf89787asdfsad87676" -BaseURL http://prowlarr:9696
-
+    PS> .\Convert-ProwlarrSupportedIndexersToMarkdownTable.ps1 -Commit 1.1.1.1 -Build "test" -AppAPIKey "asjdhfjashdf89787asdfsad87676" -AppBaseURL http://prowlarr:9696
     .EXAMPLE
-    PS> .\Convert-ProwlarrSupportedIndexersToMarkdownTable.ps1 -Commit 1.1.1.1 -Build "test" -APIKey "asjdhfjashdf89787asdfsad87676" -OutputFile "supported-indexers.md"
-
+    PS> .\Convert-ProwlarrSupportedIndexersToMarkdownTable.ps1 -Commit 1.1.1.1 -Build "test" -AppAPIKey "asjdhfjashdf89787asdfsad87676" -OutputFile "supported-indexers.md"
     .EXAMPLE
-    PS> .\Convert-ProwlarrSupportedIndexersToMarkdownTable.ps1 -Commit 1.1.1.1 -Build "test" -APIKey "asjdhfjashdf89787asdfsad87676"
+    PS> .\Convert-ProwlarrSupportedIndexersToMarkdownTable.ps1 -Commit 1.1.1.1 -AppAPIKey "asjdhfjashdf89787asdfsad87676"
 #>
 
+[CmdletBinding()]
 param (
-    [Parameter(
-        Mandatory, 
-        Position = 1
-    )]
-    [string]
-    $Commit,
-    [Parameter(
-        Mandatory, 
-        Position = 2
-    )]
-    [string]
-    $Build,
-    [Parameter(
-        Mandatory,
-        Position = 3
-    )]
-    [string]
-    $APIKey,
-    [Parameter(
-        Position = 4
-    )]
+    [Parameter(Mandatory, Position = 1)]
+    [string]$Commit,
+
+    [Parameter(Position = 2)]
+    [string]$Build,
+
+    [Parameter(Mandatory, Position = 3)]
+    [string]$AppAPIKey,
+
+    [Parameter(Position = 4)]
     [System.IO.FileInfo]
     $OutputFile = ".$([System.IO.Path]::DirectorySeparatorChar)supported-indexers.md",
```

Comment:
You've collapsed the types for all of these onto the same line as the parameter name except for OutputFile